main.cpp

```diff
@@ -16,29 +16,41 @@ video: Chapter 2 - Part 3
  
  1) Write down the names of all of the primitives available in C++ (excluding wchar_t)
  put them here: 
- 
- 
- 
- 
- 
- 
- 
- 
- 
- 
- 
+
+
+
+   * Integer 
+   * Character
+   * Boolean
+   * Floating Point
+   * Double Floating Point
+   * Void 
+   * Wide Character
```

Comment:
- [ ] please write down the C++ type name, not the english name

Reply:
Done. Changed to c++ primitive names.